resources/base/console-ui.yaml

```
@@ -289,7 +290,7 @@ spec:
         - args:
             - ui
             - --certs-dir=/tmp/certs
-          image: minio/operator:v5.0.12
+          image: minio/operator:v5.0.13
```

5.0.13 is not released yet
```suggestion
          image: minio/operator:v5.0.12
```